eo-phi-normalizer/src/Language/EO/Phi/Rules/Yaml.hs

```diff
@@ -34,9 +35,16 @@ data RuleSet = RuleSet
   }
   deriving (Generic, FromJSON, Show)
 
+data RuleContext = RuleContext
+  { global_object :: Maybe Object
+  , current_object :: Maybe Object
```

Comment:
Could they both even be specified together? Or is it one or the other?

Reply:
Allowing both is fine.